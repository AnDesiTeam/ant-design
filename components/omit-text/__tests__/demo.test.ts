import demoTest from '../../../tests/shared/demoTest';

demoTest('omit-text');
