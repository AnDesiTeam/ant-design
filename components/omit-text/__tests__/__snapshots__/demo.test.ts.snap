// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/omit-text/demo/basic.md correctly 1`] = `
<div
  class="omitText"
  style="max-width:120px"
>
  这是一段很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长的文本
</div>
`;

exports[`renders ./components/omit-text/demo/type.md correctly 1`] = `
<div
  class="omitText"
  style="max-width:120px"
>
  这是一段很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长的文本
</div>
`;

exports[`renders ./components/omit-text/demo/width.md correctly 1`] = `
<div
  class="omitText"
  style="max-width:250px"
>
  这是一段很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长的文本
</div>
`;
